tools/istio-iptables/pkg/cmd/run_test.go

```diff
@@ -215,8 +215,13 @@ func TestRulesWithTproxy(t *testing.T) {
 		"iptables -t nat -A OUTPUT -p udp --dport 53 -m owner --gid-owner 1337 -j RETURN",
 		"iptables -t nat -A OUTPUT -p udp --dport 53 -d 127.0.0.53/32 -j REDIRECT --to-port 15053",
 		"iptables -t mangle -A PREROUTING -p tcp -m mark --mark 1337 -j CONNMARK --save-mark",
+		"iptables -t mangle -A OUTPUT -p tcp -o lo -m mark --mark 1337 -j RETURN",
+		"iptables -t mangle -A OUTPUT ! -d 127.0.0.1/32 -p tcp -o lo -m owner --uid-owner 1337 -j MARK --set-mark 1338",
+		"iptables -t mangle -A OUTPUT ! -d 127.0.0.1/32 -p tcp -o lo -m owner --gid-owner 1337 -j MARK --set-mark 1338",
 		"iptables -t mangle -A OUTPUT -p tcp -m connmark --mark 1337 -j CONNMARK --restore-mark",
-		"iptables -t mangle -I ISTIO_INBOUND 1 -p tcp -m mark --mark 1337 -j RETURN",
+		"iptables -t mangle -I ISTIO_INBOUND 1 -p tcp -s 127.0.0.6/32 -i lo -j RETURN",
+		"iptables -t mangle -I ISTIO_INBOUND 2 -p tcp -i lo -m mark ! --mark  1338 -j RETURN",
```

Comment:
Since all 3 rules result in a return, the one most likely should be first.

Reply:
done